app/utils/API1811/ChargeTypes.scala

```diff
@@ -46,13 +75,24 @@ object ChargeTypes extends LoggerUtil {
     ("4710", vatInterestRepaymentsSubTrans) -> "VAT Repayment Supplement",
     ("4711", vatDebitSubTrans) -> "VAT Indirect Tax Revenue Rec",
     ("4715", vatInterestRepaymentsSubTrans) -> "VAT Statutory Interest",
+    ("4716", penaltyDebitSubTrans) -> "VAT POA Return 1st LPP",
+    ("4717", penaltyDebitSubTrans) -> "VAT POA Return 2nd LPP",
+    ("4718", penaltyDebitSubTrans) -> "VAT AA Return Charge 1st LPP",
+    ("4719", penaltyDebitSubTrans) -> "VAT AA Return Charge 2nd LPP",
     ("4720", vatDebitSubTrans) -> "VAT Central Assessment",
     ("4721", vatInterestSubTrans) -> "VAT Default Interest",
     ("4722", vatInterestSubTrans) -> "VAT Further Interest",
+    ("4723", penaltyDebitSubTrans) -> "VAT Central Assessment 1st LPP",
+    ("4724", penaltyDebitSubTrans) -> "VAT Central Assessment 2nd LPP",
     ("4725", vatInterestSubTrans) -> "VAT OA Further Interest",
     ("4726", vatInterestSubTrans) -> "VAT AA Further Interest",
     ("4727", vatInterestSubTrans) -> "VAT EC Further Interest",
     ("4728", vatInterestSubTrans) -> "VAT PA Further Interest",
+    ("4741", penaltyDebitSubTrans) -> "VAT OA 1st LPP",
```

Comment:
Quite a few charges out of order here, in the 4730 - 4750 range. I know this is just for our purposes but it did confuse me a little when reviewing.